tapdb/migrations.go

```diff
@@ -34,6 +45,35 @@ var (
 	}
 )
 
+var (
+	// ErrMigrationDowngrade is returned when a database downgrade is detected.
```

Comment:
line length over 80, fails linter now 🎉 

Not sure why this needs its own `var ( )` block either?